### ISSUE TITLE
Upgrading to PHP v5.6.13 Fails on OS X v10.10.5 Yosemite

### ISSUE THREAD

**Issue by RandomDSdevel** [Homebrew/homebrew-php#2047]
Parameter | Value
------------------ | ------------------
**OS X Version:** | v10.10.5 'Yosemite,' as provided in issue title
**Homebrew Version:** | `0.9.5 (git revision 1eba5c; last commit 2015-09-10)`
**PHP Version in use:** | v5.6.12
**Xcode Version:** | v6.4
**Output of `xcodebuild -version -sdk`:** | See below in note 1.  
**Output of `php -v`:** | See below in note 2.  
**Output of `brew upgrade -vd --build-from-source php56` within a gist** | `upgrade.log` in [this Gist](https://gist.github.com/RandomDSdevel/09f7b2c512cd3fab821b)
**Output of `brew doctor -vd` within a gist** | `doctor.log` in  [this Gist](https://gist.github.com/RandomDSdevel/09f7b2c512cd3fab821b)

Note 1:  Output of `xcodebuild -version -sdk`:  
```
MacOSX10.9.sdk - OS X 10.9 (macosx10.9)
SDKVersion: 10.9
Path: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk
PlatformVersion: 1.1
PlatformPath: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform
ProductBuildVersion: 13F34
ProductCopyright: 1983-2014 Apple Inc.
ProductName: Mac OS X
ProductUserVisibleVersion: 10.9.5
ProductVersion: 10.9.5

MacOSX10.10.sdk - OS X 10.10 (macosx10.10)
SDKVersion: 10.10
Path: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.10.sdk
PlatformVersion: 1.1
PlatformPath: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform
ProductBuildVersion: 14D125
ProductCopyright: 1983-2015 Apple Inc.
ProductName: Mac OS X
ProductUserVisibleVersion: 10.10.3
ProductVersion: 10.10.3

iPhoneOS8.4.sdk - iOS 8.4 (iphoneos8.4)
SDKVersion: 8.4
Path: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk
PlatformVersion: 8.4
PlatformPath: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform
ProductBuildVersion: 12H141
ProductCopyright: 1983-2015 Apple Inc.
ProductName: iPhone OS
ProductVersion: 8.4

iPhoneSimulator8.4.sdk - Simulator - iOS 8.4 (iphonesimulator8.4)
SDKVersion: 8.4
Path: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.4.sdk
PlatformVersion: 8.4
PlatformPath: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform
ProductBuildVersion: 12H141
ProductCopyright: 1983-2015 Apple Inc.
ProductName: iPhone OS
ProductVersion: 8.4

Xcode 6.4
Build version 6E35b
```

Note 2:  Output of `php -v`:
```
PHP 5.6.12 (cli) (built: Aug  8 2015 12:19:08) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies
```

**Comment by ablyler** [comment]
This was fixed with PR #2049.

**Comment by RandomDSdevel** [comment]
Ah, goody!
